src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -5638,4 +5653,37 @@ private Set<String> getGeneratedFeatures() {
                 generatedFeaturesFile, null, null);
         return genFeatSet;
     }
+
+    /**
+     * Create metadata when running devc mode and containers
+     * Language server then uses metadata file to connect
+     * 
+     * @throws XMLStreamException
+     * @throws IOException
+     * @throws FactoryConfigurationError
+     */
+    public void writeDevcMetadata() {
+        File metaFile = new File(serverDirectory, "liberty-dev-metadata.xml");
```

Comment:
As discussed in Slack, let's use the build directory instead of the server directory for the location of this file, and simply add the name of the server to the file name.